docs/gen-docs/kyma_alpha_create_module.md

```diff
@@ -13,26 +13,30 @@ Use this command to create a Kyma module and bundle it as an OCI image.
 Kyma modules are individual components that can be deployed into a Kyma runtime. Modules are built and distributed as OCI container images. 
 With this command, you can create such images out of a folder's contents.
 
-This command creates a component descriptor in the descriptor path (./mod as a default) and packages all the contents on the provided content path as a single layer.
-Optionally, you can create additional layers with contents in other paths.
+This command creates a component descriptor in the descriptor path (./mod as a default) and packages all the contents on the provided path as an OCI image.
+Kubebuilder projects are supported, if the path contains a kubebuilder project, it will be built and pre-defined layers will be created based on its known contents.
```

Comment:
```suggestion
Kubebuilder projects are supported. If the path contains a kubebuilder project, it will be built, and the pre-defined layers will be created based on its known contents.
```